generate.xml

```diff
@@ -107,8 +107,6 @@
         <flag substitute="icu" />
         <flag substitute="png" />
         <flag substitute="qrencode" />
-        <library name="png" />
-        <library name="qrencode" />
         <library name="bitcoin" />
```

Comment:
Good catch on the libraries - these are picked up by the dependency chaining and in other projects by package config.